_maps/configs/minutemen_chiron.json

```
@@ -0,0 +1,40 @@
+	{
+	"$schema": "https://raw.githubusercontent.com/shiptest-ss13/Shiptest/master/_maps/ship_config_schema.json",
+	"map_name": "Chrion-class Emergency Response Vessel",
+	"prefix": "CMSV",
+	"namelists": ["GENERAL", "CLIP MINUTEMEN", "NATURAL_AGGRESSIVE"],
+	"description": "The Chiron-class is a new-ish ship purpose-built for BARD. Built from hulls designed and made, but never used, during the Frontiersmen War, and equipped with a small but powerful ambulance shuttle, these hulls find new life as Search and Rescue platforms, as well as aiding CMM-BARD in disaster relief and biohazard containment.",
```

```suggestion
	"description": "The Chiron-class is a relatively new, purpose-built vessel for use by CMM-BARD. The Chiron's construction comes from hulls designed and made during the Frontiersmen War, repurposed and modernized as a disaster relief ship in response to Ryunosuke's Xenofauna Infestation. As a platform meant to adequately handle a multitude of disasters, it is equipped with numerous medical equipment, a nimble rescue subshuttle, and an armory for self-defense, but falls short of specialization at any of these aspects.",
```
i'm not sure about the "new-ish" part, considering that a frontiersmen war design would be some ~110 years old and a xenofauna war vessel would be around ~75 years old